docs/.gitignore

```diff
@@ -1,3 +1,2 @@
 release/next/api/*
-source/cli/*
```

Comment:
@malachi-constant I had to remove this in order to push the `docs/source/cli/aws_ddk.rst` file. Is this required in the .gitignore? If so, can we merge this and then put it back?

Reply:
Yes we can and should remove this now that we aren't autogenerating those source *rst files.